util/default.json

```diff
@@ -26,7 +26,7 @@
 			"display": 2
 		}
 	},
-	"defaultPermissions": "275817811008",
+	"defaultPermissions": "4398046511103",
```

Comment:
Asking for excessive permissions (literally every existing permission)

Besides the point that bots can't even ask for administrator permissions, bot should never request more permissions than nessesary.